README.md

```diff
@@ -2,55 +2,32 @@
 
 # Project Name
 
+Survey Form
```

Comment:
A more descriptive name should have been given to the survey form e.g Terrestrial and Aquatic Wildlife Survey form

Reply:
- [ ] [REQUIRED] Please, always try to mention the reason for your requested chagnes. Because it helps the students learn more.